core/src/main/java/org/infinispan/conflict/impl/DefaultConflictManager.java

```diff
@@ -90,8 +90,8 @@
    @Inject Configuration cacheConfiguration;
    @Inject CommandsFactory commandsFactory;
    @Inject DistributionManager distributionManager;
-   @Inject @ComponentName(ASYNC_TRANSPORT_EXECUTOR)
-   ExecutorService transportExecutor;
+   @Inject @ComponentName(PERSISTENCE_EXECUTOR)
+   ExecutorService persistenceExecutor;
```

Comment:
Do we need a different name for this executor if it's going to be used for non-peristence related executions?

Reply:
It is really the blocking executor at this point. I will be having a PR that renames and moves all these over soon.